packages/css/stories/Button.js

```diff
@@ -0,0 +1,19 @@
+import '../src/components/button.css';
+
+export const createButton = ({
+  mode = 'secondary',
+  size = 'medium',
+  disabled = false,
+  label,
+  onClick,
+}) => {
+  const btn = document.createElement('button');
+  btn.type = 'button';
+  btn.innerText = label;
+  btn.addEventListener('click', onClick);
+
+  btn.className = ['button-base', 'button-master', mode, size].join(' ');
+  btn.disabled = disabled;
+
+  return btn;
+};
```

Comment:
Can we generate the template using template literals as suggested in the story file

```js
  // You can either use a function to create DOM elements or use a plain html string!
  // return `<div>${label}</div>`;
```

Reply:
I don't think it will look simpler for the `button` case, as we need to handle stuff like `disabled` and `onclick`.